models/Device.js

```diff
@@ -91,7 +130,8 @@ module.exports = function(sequelize, DataTypes) {
     return true;
   };
 
-  var onlyUsersDevicesMatching = async function (user, conditions = null, includeData = null) {
+  /* .. */
```

Comment:
I like to have a visual separator between functions because it helps with navigation; it was also sort of a deliberate TBD to eventually get function headers for each function. Anyway, I can remove them.